modules/navier_stokes/doc/content/bib/navier_stokes.bib

```diff
@@ -119,17 +119,6 @@ @article{kurganov2000new
   publisher={Elsevier}
 }
 
-@article{greenshields2010implementation,
```

Comment:
I tried this and it used to error out. Let s see what the doc target says

Reply:
Yea I got a warning when building `navier_stokes` documentation *because* this existed. So it will be unfortunate haha if you get warned one way and error the other